### DISCUSSION TITLE
[Bridge/Nuxt3]: Use Directives in SSR.

### DISCUSSION THREAD

**Discussion by ThornWalli** [nuxt/nuxt#16311] [Questions]
Hello,

I wanted to ask how the definition of Vue Directives in SSR is solved from Bridge on.

The Nuxt 2 implementation doesn't seem to work anymore. 
https://nuxtjs.org/docs/configuration-glossary/configuration-render/#bundlerenderer

Unfortunately I can't find anything useful on this topic :(

**Comment by TheAlexLichter** [comment]
Hey! feel free to check out https://github.com/nuxt/framework/issues/1659 + linked issues.

**Comment by ThornWalli** [comment]
I had created another issue on this topic. 

First away from the bridge, now directly on nuxt 3. There the Directiven integration works well except for a strange behavior of `getSSRProps`.

There I now have the problem that values that are exposed in the component do not appear in the instance in `getSSRProps`. On the client side the `instance` is filled.

Is this a `nuxt` or `vue` problem?

https://github.com/nuxt/framework/issues/8862
